packit/api.py

```diff
@@ -2139,6 +2143,107 @@ def run_osh_build(
 
         return cmd_result.stdout
 
+    def create_obs_project(
+        self,
+        project: str,
+        targets: str,
+        owner: Optional[str],
+        package_config: PackageConfig,
+        description: Optional[str],
+    ):
+        conf.get_config()
+        owner = (
+            owner or conf.config["api_host_options"][self.obs_helper._API_URL]["user"]
+        )
+        prj_name = project or f"home:{owner}:packit"
+
+        targets_list = targets.split(",")
+        for target in targets_list:
+            if target in DEPRECATED_TARGET_MAP:
+                logger.warning(
+                    f"Target '{target}' is deprecated. "
+                    f"Please use '{DEPRECATED_TARGET_MAP[target]}' instead.",
+                )
+
+        prj_meta = self.obs_helper.targets_to_project_meta(
+            targets=targets_list,
+            owner=owner,
+            prj_name=prj_name,
+            description=description,
+        )
+
+        logger.info(f"Using OBS project name = {prj_name}")
+
+        url = core.makeurl(self.obs_helper._API_URL, ["source", prj_name, "_meta"])
+        mf = core.metafile(url, ET.tostring(prj_meta))
+        mf.sync()
```

Comment:
Same as above, it would help me (and hopefully others..;) a lot if we could have more clear names.